app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser_inventory_object.rb

```diff
@@ -59,6 +59,11 @@ def parse_snapshot(snap)
     }
   end
 
+  # Overridden helper methods, we should put them in helper once we get rid of old refresh
+  def get_from_tags(resource, item)
+    (resource['tags'] || []).detect { |tag, _| tag['key'].downcase == item.to_s.downcase }.try(:[], 'value')
+  end
+
```

Comment:
It hurt very much copying it once more. I was thinking about using a different name. For example, I thought about renaming the existing `get_from_tags` in the helpers module and using this one instead, but just couldn't find a proper name (`get_from_tags_legacy` or `get_from_tags_refresh_parser` didn't sound ok to me).

Any idea?